go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -398,6 +398,79 @@ func TestMissingTables(t *testing.T) {
 	runCases(t, filter, testcases, startPos, nil)
 }
 
+// TestInternalTables tests streaming of sidecar db tables.
```

Comment:
This comment for the function name is another clue that it might be better/clearer to call them SidecarDBTables rather than InternalTables.